examples/ZMQPP/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 3.2)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-ZMQPP)
+
+hunter_add_package(ZMQPP)
+
+find_package(ZMQPP CONFIG REQUIRED)
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
```

Comment:
I think it will also be needed to patch `CMakeLists.txt` of `zmqpp` to remove the lines at https://github.com/hunter-packages/zmqpp/blob/hunter/CMakeLists.txt#L17-L21

```cmake
# prepare C++11
set(CMAKE_CXX_FLAGS "-std=c++11 ${CMAKE_CXX_FLAGS}")

# show all warnings
set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -pedantic -Wextra")
```

So it takes whatever it set by the toolchain.

I guess when hunter moves to CMake 3.1, ideally we would add
```cmake
set_target_properties(... PROPERTIES
    CXX_STANDARD 11
    CXX_STANDARD_REQUIRED ON
)
```